### DISCUSSION TITLE
RStudio and GitHub credentials

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1107] [Q&A] [answered]
How do you provide RStudio GitHub credentials?

**Comment by sr320** [comment, marked as answer]
Step 1 - Get a PAT
(https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)

Step 2 - Store credentials - I used "gitcreds" package.

(https://happygitwithr.com/credential-caching.html#store-your-credential)

**Comment by grace-ac** [comment]
Link to "gitcreds" doesn't work.
